### ISSUE TITLE
Some users can not get

### ISSUE THREAD

**Issue by kgtkr** [QuietMisdreavus/twitter-rs#60]
```rs
println!("{:?}", core.run(egg_mode::user::show("GJ_boy1", &token)));
// Err(DeserializeError(Error("invalid type: null, expected a string", line: 1, column: 351)))
```
```json
{"id":251121418,"id_str":"251121418","name":"\u30ef\u30af\u30dc\uff11","screen_name":"GJ_boy1","location":"\u4eac\u90fd","profile_location":null,"description":"911S \/ VRSCAW \/ \u30b7\u30fc\u30de \/ \u30c7\u30a3\u30a2\u30d9\u30eb","url":"http:\/\/www.disney.co.jp\/","entities":{"url":{"urls":[{"url":"http:\/\/www.disney.co.jp\/","expanded_url":null,"indices":[0,24]}]},"description":{"urls":[]}},"protected":false,"followers_count":5833,"friends_count":564,"listed_count":598,"created_at":"Sat Feb 12 13:47:39 +0000 2011","favourites_count":394340,"utc_offset":null,"time_zone":null,"geo_enabled":false,"verified":false,"statuses_count":3261,"lang":null,"status":{"created_at":"Tue May 28 10:37:32 +0000 2019","id":1133321401748709377,"id_str":"1133321401748709377","text":"\u30d7\u30e9\u30c3\u30c8\u30d5\u30a9\u30fc\u30e0\u306f\u5171\u901a\u5316\u3057\u3066\u30b3\u30b9\u30c8\u30c0\u30a6\u30f3\u3001\u5148\u9032\u6280\u8853\u3084\u30e1\u30fc\u30bf\u30fc\u6db2\u6676\u306f\u5168\u3066\u30aa\u30d7\u30b7\u30e7\u30f3\u3001\u305d\u308c\u3067\u3082\u6700\u4f4e\u30b0\u30ec\u30fc\u30c9\u306e\u65e5\u672c\u4ed5\u69d8\u306f\u4e57\u308a\u51fa\u3057350\u4e07\u3066\u30de\u30b8\u30672\u30b7\u30ea\u30fc\u30ba\u30b0\u30e9\u30f3\u30c9\u30c4\u30a2\u30e9\u30fc\u8cb7\u3063\u305f\u3001\u8133\u306b\u969c\u5bb3\u3092\u304a\u6301\u3061\u306e\u65b9\u5411\u3051\u306e\u8eca\u306b\u306a\u308b\u3093\u3084\u308d\u306a\u2026\u2026","truncated":false,"entities":{"hashtags":[],"symbols":[],"user_mentions":[],"urls":[]},"source":"\u003ca href=\"http:\/\/tapbots.com\/tweetbot\" rel=\"nofollow\"\u003eTweetbot for i\u039fS\u003c\/a\u003e","in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"geo":null,"coordinates":null,"place":null,"contributors":null,"is_quote_status":false,"retweet_count":0,"favorite_count":5,"favorited":false,"retweeted":false,"lang":"ja"},"contributors_enabled":false,"is_translator":false,"is_translation_enabled":false,"profile_background_color":"FFFFFF","profile_background_image_url":"http:\/\/abs.twimg.com\/images\/themes\/theme1\/bg.png","profile_background_image_url_https":"https:\/\/abs.twimg.com\/images\/themes\/theme1\/bg.png","profile_background_tile":false,"profile_image_url":"http:\/\/pbs.twimg.com\/profile_images\/1120737070685638656\/9yJF2CCN_normal.png","profile_image_url_https":"https:\/\/pbs.twimg.com\/profile_images\/1120737070685638656\/9yJF2CCN_normal.png","profile_banner_url":"https:\/\/pbs.twimg.com\/profile_banners\/251121418\/1551628385","profile_link_color":"E81C4F","profile_sidebar_border_color":"FFFFFF","profile_sidebar_fill_color":"DDEEF6","profile_text_color":"333333","profile_use_background_image":true,"has_extended_profile":false,"default_profile":false,"default_profile_image":false,"following":null,"follow_request_sent":null,"notifications":null,"translator_type":"none"}
```

**Comment by kkostov** [comment]
I am experiencing a similar problem with the `show()` method. 
In my case, the error is the following:

```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: MissingValue("lang")'
```

I believe this issue is the same as #59 which seems to be addressed by #57.

**Comment by adwhit** [comment]
@kgtkr I have merged #57, has that fixed your problem?

**Comment by kgtkr** [comment]
Thank you
solved
